datadog_lambda/wrapper.py

```diff
@@ -102,34 +105,42 @@ def __init__(self, func):
 
     def __call__(self, event, context, **kwargs):
         """Executes when the wrapped function gets called"""
-        self._before(event, context)
+        trigger_tags = extract_trigger_tags(event, context)
```

Comment:
Nits -- I think it's better to put `trigger_tags` and `response` onto `self` (similar to `self.span`), so you don't have to pass them to `_before` or `_after`. TBH, we could have apply the same pattern to `event` and `context`, but they seem to be important enough to standout.

Reply:
Much cleaner now 👍